#2669 Fixed XSS vulnerabilities in graphical view components:

Limraj opened the pull request:
- Fixed Simple Point: Name, Point name override, Style attribute, Display controls, Background color;
- Fixed Simple compound: Name, Background colour, Lead point, Sub point X;

Limraj committed:
#2669 Fixed XSS vulnerabilities in graphical view components:
- Fixed Simple Point: Name, Point name override, Style attribute, Display controls, Background color;
- Fixed Simple compound: Name, Background colour, Lead point, Sub point X;

Patrykb0802 reviewed (commented):
Fixed Simple Point: 

Point value is showing like: ![<span>{value}</span>]() --> error

 - Name ---> ok, point selected with name of !["><img src=x onerror=alert(document.location)>]() is not causing any issues
 - Point name override --> ok:
![image](https://github.com/user-attachments/assets/19823ada-944e-489d-977a-644521168f36)

- Style attribute ---> seems to not being escaped:

Set value: !["><img src=x onerror=alert(document.location)>]()

![image](https://github.com/user-attachments/assets/6e14150c-7c34-469f-91d7-d0a606507437)

Display controls ---> ok, content is escaped:
![image](https://github.com/user-attachments/assets/597d7b12-66e6-408d-8425-26b2a061c2d1)

Background color ---> seems to not being escaped:

Set value: !["><img src=x onerror=alert(document.location)>]()

![image](https://github.com/user-attachments/assets/bc5ad15d-b334-455a-8cfd-4ea494fb9f56)


 - Fixed Simple compound: 

  - Name --> ok:
![image](https://github.com/user-attachments/assets/3dda81d5-e402-403a-a807-0dd4b4c014bf)

  
  - Background colour --> setting value with potential XSS and saving it --> no error --> saving graphical view and on read mode alert shows up one time:
![image](https://github.com/user-attachments/assets/a525a8eb-478a-44ae-94a8-0d9ed663376c)


  - Lead point --> ok, no alerts when data point with such name is selected:
  
  - Sub point X --> ok, no alerts when data point with such name is selected:
![image](https://github.com/user-attachments/assets/ce2e0a3d-4dfb-4a05-8983-08b5310d48a3)

Limraj committed:
#2669 Fixed XSS vulnerabilities in graphical view components:
- Fixed: <span>value</span>;
- Fixed: Background color, Style attribute;
- Fixed value alphanumeric type: added properties: datapoint.type.alphanumeric.escaped default false;

Patrykb0802 reviewed (approved):
Simple point works fine now:
All fields are escaped
![image](https://github.com/user-attachments/assets/ff4a7994-371c-4a10-ba30-624626aaa471)


Simple compound works fine:
![image](https://github.com/user-attachments/assets/524e0474-ed9e-4e35-a64d-6ff11ffdca65)

no pop ups/alerts are occuring
